create_tarball.sh

```diff
@@ -21,7 +22,18 @@ os="linux"
 source ${base_dir}/init/eessi_defaults
 cvmfs_repo=${EESSI_CVMFS_REPO}
 
-software_dir="${cvmfs_repo}/versions/${eessi_version}/software/${os}/${cpu_arch_subdir}"
+# install to $eessi_version/$eessi_dev_project if defined, otherwise install
+# to $eessi_version
+install_prefix_dir=${eessi_version}${eessi_dev_project:+/$eessi_dev_project}
+echo "Setting Install prefix directory to: ${install_prefix_dir}"
+
+if [ -z ${install_prefix_dir} ]; then
+    echo "\${install_prefix_dir} is empty?!" >&2
+    exit 2
+fi
```

Comment:
What is this supposed to match? It doesn't look like it could match the new location `/cvmfs/dev.eessi.io/${eessi_dev_project}/versions/...`

From the comment, shouldn't it be `$eessi_dev_project/versions/$eessi_version`

Reply:
What about keeping the code as was, but updating line 23 such that
```
cvmfs_repo=${EESSI_CVMFS_REPO}${eessi_dev_project:+/$eessi_dev_project}
```

Reply:
Agreed! Changed in acb896a163118d103e62e65566e052ce05c8ccc6